internal/rbd/rbd_journal.go

```diff
@@ -598,16 +603,23 @@ func RegenerateJournal(
 	if err != nil {
 		return "", err
 	}
+
 	if imageData != nil {
 		rbdVol.ReservedID = imageData.ImageUUID
 		rbdVol.ImageID = imageData.ImageAttributes.ImageID
 		rbdVol.Owner = imageData.ImageAttributes.Owner
+		rbdVol.RbdImageName = imageData.ImageAttributes.ImageName
 		if rbdVol.ImageID == "" {
 			err = rbdVol.storeImageID(ctx, j)
 			if err != nil {
 				return "", err
 			}
 		}
+		// Update Metadata on reattach of the same old PV
+		err = rbdVol.setVolumeDetailsAsMetadata(volumeAttributes, claimName, rbdVol.Owner, setMetadata)
```

Comment:
owner is already member of rbdVol. we don't need to sent it as input to `setVolumeDetailsAsMetadata`

Reply:
call this only if setMetadata is set here only we don't need to pass that as a argument?